### PR TITLE
Employment Tribunals changes for path to live of citizen based App

### PR DESCRIPTION
**Pull request by petermoores-version1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-2081

### Change description ###
Employment Tribunals changes for path to live of citizen based App. Includes shuttering changes.
Includes new app yet to be developed.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: components/prod/respond-employment-tribunals.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+data "local_file" "claim-et-ra-config" {
```
Comment (chris-pateman) [inline components/prod/respond-employment-tribunals.tf:1]: Should this be `respond-et-ra-config`

File: environments/prod/claim-et-sya-hmcts-net.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ns:
```
Comment (chris-pateman) [inline environments/prod/claim-et-sya-hmcts-net.yml:1]: these empty ones need `[]` next to them e.g. `ns: []`

**Review by chris-pateman — COMMENTED** [review]
Some minor updates needed

File: environments/prod/claim-et-sya-hmcts-net.yml (hunk 2 above)
Comment (hsjhita) [reply environments/prod/claim-et-sya-hmcts-net.yml]: Refactored

**Review by chris-pateman — APPROVED** [review]
LGTM

**Pull request merged**